common/rpc/outbounds_test.go

```diff
@@ -98,20 +99,22 @@ func TestPublicClientOutbound(t *testing.T) {
 	_, err := newPublicClientOutbound(&config.Config{})
 	require.EqualError(t, err, "need to provide an endpoint config for PublicClient")
 
-	builder, err := newPublicClientOutbound(makeConfig("localhost:1234", false, ""))
+	builder, err := newPublicClientOutbound(makeConfig("localhost:1234", "tchannel", false, ""))
 	require.NoError(t, err)
 	require.NotNil(t, builder)
 	require.Equal(t, "localhost:1234", builder.address)
 	require.Equal(t, nil, builder.authMiddleware)
+	require.False(t, builder.isGrpc)
 
-	builder, err = newPublicClientOutbound(makeConfig("localhost:1234", true, "invalid"))
+	builder, err = newPublicClientOutbound(makeConfig("localhost:1234", "tchannel", true, "invalid"))
 	require.EqualError(t, err, "create AuthProvider: invalid private key path invalid")
 
```

Comment:
Nit: As you added `require.True(t, builder.isGrpc)`, maybe add False check here?
```suggestion
	require.False(t, builder.isGrpc)
```